src/parser.jl

```diff
@@ -444,7 +444,7 @@ function parse_comparison(ps::ParseState, ts::TokenStream, ops=INEQ_OPS)
     isfirst = true
     while true
         t = peek_token(ps, ts)
-        if !(¬t in ops || ¬t === :in)
+        if !(¬t in ops || ¬t === :in || (VERSION >= v"0.6.0-dev.1471" && ¬t === :isa))
```

Comment:
Sorry, I guess the version should have a + instead of a ., i.e. `v"0.6.0-dev+1741"`. That was my bad.